restore.sh

```diff
@@ -3,5 +3,4 @@
 set -euo pipefail
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-repo_root="$DIR/../.."
-"$repo_root/build.sh" --all --restore --no-build "$@"
+"$DIR/build.sh" --all --restore --no-build "$@"
```

Comment:
@natemcmaster Though this works, the script is different from the one in aspnet/Extensions any idea which one we should be using?

Reply:
aspnet/Extensions's script is different because it is using a different build system, Arcade. We haven't converted this repo yet. (#7280)